internal/output/format.go

```diff
@@ -23,12 +26,46 @@ func formatCost(currency string, d *decimal.Decimal) string {
 	if d == nil {
 		return "-"
 	}
+
+	if os.Getenv("INFRACOST_CURRENCY_FORMAT") != ""{
```

Comment:
This should be loaded as an [`envconfig`](https://github.com/infracost/infracost/blob/5ea3f9d6de8737d81930c836ea1f0fa60c379c83/internal/config/config.go#L81) like the INFRACOST_CURRENCY env.

Reply:
I made some changes, not sure if that is enough, I have just started coding in go :)